lib/azure/armrest/resource_provider_service.rb

```diff
@@ -53,7 +54,8 @@ def list
       def get(namespace)
         url = build_url(namespace)
         response = rest_get(url)
-        JSON.parse(response.body)
+        #JSON.parse(response.body)
+        Azure::Armrest::ResourceProvider.new(response.body)
```

Comment:
This shouldn't be in there, sorry. I must have been tinkering and left it in by mistake.

For list methods, we'll need to .map the results.